sdk/privatedns/Azure.ResourceManager.PrivateDns/src/Generated/PrivateZoneData.cs

```diff
@@ -56,7 +56,7 @@ internal PrivateZoneData(ResourceIdentifier id, string name, ResourceType resour
         /// <summary> The maximum number of record sets that can be created in this Private DNS zone. This is a read-only property and any attempt to set this value will be ignored. </summary>
         public long? MaxNumberOfRecordSets { get; }
```

Comment:
It seems that we changed `RecordSets` to `Records` in other places, do we also need to change this one?